tests/test_func.py

```diff
@@ -94,6 +94,42 @@ def test_func_on_more_than_one_table(db: gp.Database):
     assert "Cannot pass arguments from more than one tables" == str(exc_info.value)
 
 
+def test_create_func_optional_params(db: gp.Database):
+    @gp.create_function
+    def multiply(a: int, b: int) -> int:
+        return a * b
+
+    @gp.create_function(replace_if_exists=True)
```

Comment:
Is this test correct? I tried to change the `replace_if_exists` to `False`, and the test can still pass.

Reply:
Corrected